karma.conf.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function(config) {
+  config.set({
+      frameworks: ["jasmine", "karma-typescript"],
+      files: [
+          "src/**/*.ts" 
+      ],
+      preprocessors: {
+          "**/*.ts": "karma-typescript" 
+      },
+      reporters: ["progress", "karma-typescript"],
+      browsers: ["Chrome"],
```

Comment:
Can we set to use ChromeHeadless? Great for running in CI. See cloudportal-frontend

Reply:
sure